src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -81,3 +82,13 @@ def check(self, token):
         return {
             'login_url': self._get_login_url()
         }
+
+    @RESTController.Collection('POST')
+    def getNeedChangePassword(self, username):
```

Comment:
Looks like the method name should be written in `snake_case` rather then `lowerCamelCase`.